src/client/pfs/pfs.proto

```diff
@@ -169,6 +172,7 @@ message ListCommitRequest {
   CommitType commit_type = 2;
   repeated Commit from_commit = 3;
   bool block = 4;
+  bool include_cancelled = 5;
```

Comment:
Let's call this just `cancelled`, it seems implicit that we're specifying what type of object we want.

Also this is very nitpicky but could we put it above `block`? In my mind `repo`, `commit_type`, `from_commit` and `cancelled` all describe what you're getting while `block` describes how you're getting it and it makes sense to me to organize it like that.